astronomer/providers/snowflake/operators/snowflake.py

```diff
@@ -224,183 +219,20 @@ def execute_complete(self, context: Context, event: dict[str, str | list[str]] |
             raise AirflowException("Did not receive valid event from the trigerrer")
 
 
-class SnowflakeSqlApiOperatorAsync(SnowflakeOperator):
+class SnowflakeSqlApiOperatorAsync(SnowflakeSqlApiOperator):
+    """
+    This class is deprecated and will be removed in 2.0.0.
+    Use :class: `~airflow.providers.snowflake.operators.snowflake.SnowflakeSqlApiOperator` instead.
```

Comment:
Let's also mention here that they need to pass deferrable param as True like mentioned below.
This will come in the docs and would be nice to tell them what is the deferrable alternative for this deprecated operator.

Reply:
done